test/e2e/data/cluster-templates/docker-rke2.yaml

```diff
@@ -48,13 +50,18 @@ metadata:
 spec:
   replicas: ${CONTROL_PLANE_MACHINE_COUNT}
   version: ${RKE2_VERSION}
-  registrationMethod: internal-first
   rolloutStrategy:
     rollingUpdate:
       maxSurge: 1
     type: RollingUpdate
   serverConfig:
+    cni: ${RKE2_CNI}
+    kubeAPIServer:
+      extraArgs:
+      - --anonymous-auth=true
```

Comment:
Do I understand this correctly, that we authorize anonymous connection here?

Reply:
I documented the reasoning for this during upgrading to 1.9 on CAPRKE2 provider. This is a large issue, as load balancer were still treating API sever as healthy when ETCD was down due to removal of the member and local endpoint usage: https://github.com/rancher/cluster-api-provider-rke2/issues/530

Reply:
Thanks for explanation, it was non-blocking question.